packages/ui/src/working-groups/components/SelectWorkingGroupOpening/OptionsListWorkingGroupOpening.tsx

```diff
@@ -13,13 +16,13 @@ interface Props {
   onChange: (option: WorkingGroupOpening) => void
 }
 
-export const OptionsListWorkingGroupOpening = React.memo(({ allOpenings, onChange }: Props) => (
-  <OptionsListComponent>
+export const OptionsListWorkingGroupOpening = React.memo(({ allOpenings, onChange, ...props }: Props) => (
+  <OptionsListComponent {...props}>
```

Comment:
Here also the `className` would be enough to extend this component style but I don't see this done anywhere. So I think the change should just be reverted:
```suggestion
export const OptionsListWorkingGroupOpening = React.memo(({ allOpenings, onChange }: Props) => (
  <OptionsListComponent>
```